src/components/Tags/Tags.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div v-if="tags.length > 0">
+      <Tag v-for="tag in tags"
+           :key="tag"
+           :icon="icon"
+           :disabled="disabled"
+           :outlined="outlined"
+           :hovered="hovered"
+           :elevated="elevated"
+           :flat="flat"
+           :color="color"
+           class="mr-1"
+      >{{ tag }}</Tag>
```

Comment:
Random fact, you can do this if you wanted

```suggestion
      <Tag v-for="tag in tags" v-bind="{tag, icon, disabled, outlined, hovered, elevated, flat, color}" class="mr-1">{{ tag }}</Tag>```